msgraphgocore_test/graph_odata_query_handler_test.go

```diff
@@ -1,4 +1,4 @@
-package msgraphgocore
+package msgraphgocore_test
```

Comment:
I'm curious to why you changed the package name here? this could be the reason why the code coverage metric is now 0

Reply:
It's convention to have test files in a packagename_test package.

Reply:
ah, I didn't know that. would you mind taking the opportunity to change the package name in the other existing package files please?

Reply:
**msgraph-sdk-go-core** only has one package right? Or do you mean **msgraph-sdk-go**?

Reply:
correct, a single package, this might change in the future though, but it's so lightweight I don't see a necessity for another package. However according to the sdk design specs, the handlers should be in a middleware package and the page iterator in a tasks package, we should probably fix that as well.

Reply:
Yeah I thought about that but like you said it too lightweight to be necessary.

Reply:
Resolving this, will make changes if necessary.